experiments/physical_design_with_on_the_fly_gate_design/on_the_fly_sidb_circuit_design_on_defective_surface.cpp

```diff
@@ -146,35 +146,33 @@ int main()  // NOLINT
         params.exact_design_parameters.upper_bound_y = 30;         // 12 x 31 tiles
         params.exact_design_parameters.timeout       = 3'600'000;  // 1h in ms
 
-        params.sidb_on_the_fly_gate_library_parameters.defect_surface     = surface_lattice;
+        params.sidb_on_the_fly_gate_library_parameters.defect_surface     = std::optional{surface_lattice};
```

Comment:
The explicit `std::optional` constructor should usually not be necessary

Reply:
Thanks, I will double-check it.